src/zkevm_specs/evm/execution/extcodecopy.py

```diff
@@ -0,0 +1,60 @@
+from ..instruction import Instruction, Transition
+from ...util.param import EXTRA_GAS_COST_ACCOUNT_COLD_ACCESS
+from ...util import N_BYTES_MEMORY_ADDRESS, FQ
+from ..table import AccountFieldTag, CallContextFieldTag, CopyDataTypeTag
+
+
+def extcodecopy(instruction: Instruction):
+    opcode = instruction.opcode_lookup(True)
+
+    address_word, memory_offset_word, code_offset_word, size_word = (
+        instruction.stack_pop(),
```

Comment:
```suggestion
        instruction.rlc_to_fq(instruction.stack_pop(), N_BYTES_MEMORY_ADDRESS)
```

Reply:
Maybe the constant should be `N_BYTES_ACCOUNT_ADDRESS` instead of `N_BYTES_MEMORY_ADDRESS` I guess, since the address length is 20 bytes. Would you check for this? Thank you :)

Reply:
Ohh yes my bad... It should be Account. That's why it wasn't addressed! 

Good catch!!